web-ui/view/template_v1.ml

```diff
@@ -0,0 +1,69 @@
+open Tyxml.Html
+
+let html_to_string = Fmt.to_to_string (pp ())
+
+let head =
+  head
+    (title (txt "OCaml-CI"))
+    [
+      meta ~a:[ a_charset "UTF-8" ] ();
+      meta ~a:[ a_http_equiv "X-UA-Compatible"; a_content "IE=edge" ] ();
+      meta
+        ~a:
+          [
+            a_name "viewport"; a_content "width=device-width, initial-scale=1.0";
+          ]
+        ();
+      script
+        ~a:
+          [
+            a_defer (); a_src "https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js";
+          ]
+        (txt "");
+      link ~rel:[ `Stylesheet ]
+        ~href:
+          "https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap"
```

Comment:
@tmattio I remember that you suggested downloading the font if we want to use them instead of using the CDN, but I can't remember why.  Could you give us a refresh about it?

Reply:
Essentially for user privacy: users might not want to have to send query to Google (which most likely run analytics on the requests)

We're vendoring everything in ocaml.org: https://github.com/ocaml/ocaml.org/tree/main/asset/vendors

Reply:
Is there a real need for external fonts? Why not use [system-ui value for font-family](https://caniuse.com/font-family-system-ui)?